src/Utilities/Compiler/Lightup/LightupHelpers.cs

```diff
@@ -7,6 +7,8 @@
 using System.Reflection;
 using Microsoft.CodeAnalysis;
 
+#pragma warning disable CA2201 // Do not raise reserved exception types - 'throw new NullReferenceException()' is used in this file
```

Comment:
Would be nice to be able to exclude all rules on a given folder/namespace (Lightup folder).

Reply:
Good idea, let me add an editorconfig to the folder.

Reply:
I forgot about that xD I was already thinking about some exclusion pattern. I like the simple solution :)